io.github.mrstickypiston.stickyhours.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.mrstickypiston.stickyhours
+runtime: org.gnome.Platform
+runtime-version: '47'
```

Comment:
Do you actually need the gnome runtime ?

Is this for pygobject ? Since that is in freedesktop 24.08, all of its build dependencies are there too. So it should work.

Reply:
it looks like it works.